cli/tests/activate.bats

```diff
@@ -69,7 +69,8 @@ project_teardown() {
 setup() {
   common_test_setup
   setup_isolated_flox # concurrent pkgdb database creation
-  project_setup
```

Comment:
Did this move to the individual tests because we need to control during `flox init` whether we get a v0 or v1 manifest?

Reply:
Yeah - we can move it back to `setup` after we drop non-catalog tests